zul/src/org/zkoss/zul/Slider.java

```diff
@@ -458,6 +462,48 @@ public void setRange(double minpos, double maxpos) {
 		setMaxpos(maxpos);
 	}
 
+	/** Sets the angleArc.
+	 * <p>Default : 360.0
+	 * @param angleArc is the angle of the knob slider. The unit is degrees (360° for a circle).
+	 * @since 8.6.0
+	 */
+	public void setAngleArc (Double angleArc) {
```

Comment:
1. `public void setAngleArc(Double angleArc) {` A white space between Arc and (
2. double, unless you need null
3. Do some range check `0 <= value <= 360`

Reply:
ok